tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/types/TBfloat16.java

```diff
@@ -54,7 +50,7 @@
   static final String NAME = "BFLOAT16";
 
   /** Type metadata */
-  DataType<TBfloat16> DTYPE = DataType.create(NAME, 14, 2, TBfloat16Impl::mapTensor);
+  DataType<TBfloat16> DTYPE = DataType.create(NAME, 14, 2, new TBfloat16Mapper());
```

Comment:
Should we explicitly put `static final` on here? The `NAME` field is marked as such, and I spent a while looking at this line before I remembered that fields are always static and final on interfaces.

Reply:
We could but this `DataType` class is going to disappear in my next (and last?) PR about the tensor type refactoring so let's leave it as is for now.

Reply:
Sure.